kodilla-stream/src/main/java/com/kodilla/stream/array/ArrayOperations.java

```diff
@@ -0,0 +1,10 @@
+package com.kodilla.stream.array;
+
+import java.util.stream.IntStream;
+
+interface ArrayOperations {
+    static double getAverage(int[] numbers) {
+        IntStream.range(0, numbers.length).map(n -> numbers[n]).forEach(System.out::println);
```

Comment:
Do czego Ci potrzebna ta linia? Po to piszesz testy, żeby nie wypisywać takich rzeczy na konsolę.

Jeśli jednak uważasz, że jest to niezbędne, to podpowiedź - w streamie jest metoda `peek()`, wtedy możesz to wszystko zawrzeć w jednym streamie:
```
IntStream.range(...)
   .map(...)
   .peek(...)
   .average()
   .getAsDouble()
```

Wywołania łańcuchowe czytelniej się czyta, gdy każde kolejne wywołanie jest w nowej linii.

Reply:
Hej. W sumie to nie zrozumialem zadania :) pisalo, ze metoda getAverage powinna:
- przy pomocy strumienia IntStream oraz metody range(int startInclusive, int endExclusive) wyświetlić kolejne elementy tablicy - i myslalem ze chodzi o wyswietlenie kolejnych elementow tablicy - doslownie.  Wywalam, ale dobrze wiedziec ze jest metoda peek.